apps/backoffice-v2/src/lib/blocks/components/AmlBlock/utils/aml-adapter.ts

```diff
@@ -31,6 +33,14 @@ export const AmlSchema = z.object({
 
 export type TAml = z.infer<typeof AmlSchema>;
 
+const calculateEntry = (sourceInfo: z.infer<typeof SourceInfoSchema>) => {
```

Comment:
Please rename, there are no calculations in this function. You can use process / handle / etc.